string       colour     # colour label
HSVThreshold h          # hue
HSVThreshold s          # saturation
